ffi/README.md

```diff
@@ -4,13 +4,21 @@ FFI wrapper for iotics-identity-go
 
 ## Build
 
-`make compile`
+With `make` installed you can compile for the current platform with
+`make compile`. The script will detect the OS and ARCH and build for your system.
 
-The target should determine the platform and architecture of the current OS and build the binary library accordingly.
+If you want to override the target os (the `GOOS` variable), define `$OS_FLAG` to the desired value.
+For example if you want to compile for an iOS target run
+`env OS_FLAG=ios make compile`
```

Comment:
OS_FLAG=Darwin

Reply:
technically `ios` seems more correct. Although we don't yet have tags in code to include/exclude ios, we might, See 
https://pkg.go.dev/cmd/go

> Using GOOS=ios matches build tags and files as for GOOS=darwin in addition to ios tags and files.